client/flutter/lib/pages/home_page.dart

```diff
@@ -273,7 +223,8 @@ class _MenuGrid extends StatelessWidget {
                     children: <Widget>[
                       IntrinsicHeight(
                         child: _MenuButton(
-                          onTap: () => _openLatestNumbers(context),
+                          onTap: () => logAnalyticsEvent('LatestNumbers'),
+                          page: LatestNumbers(),
                           color: Color(0xff1A458E),
```

Comment:
This colour is repeated below. Should be extracted to common const variable

Reply:
I suggest we extract all the colours in the app in a separate PR and stick them in one place

Reply:
Yeah, this is an issue across the whole app.

Reply:
Since these lines were not modified by this PR, they should not be refactored in this PR.